api/admin/controller/collection_settings.py

```diff
@@ -28,6 +28,27 @@ def _get_collection_protocols(self):
         protocols = super(CollectionSettingsController, self)._get_collection_protocols(
             self.PROVIDER_APIS
         )
+
+        # dedupe and only keep the latest SETTINGS
+        # this will allow child objects to overwrite
+        # parent settings with the same key
+        # This relies on the fact that child settings
+        # are added after parent settings as such
+        # `SETTINGS + <configuration>.to_settings()`
+        for protocol in protocols:
+            if "settings" not in protocol:
+                continue
+            _found_settings = dict()
+            for ix, setting in enumerate(protocol["settings"]):
+                _key = setting["key"]
+                _found_settings[_key] = ix
+            _settings = []
+            # Go through the dict items and only use the latest found settings
+            # for any given key
+            for _, v in _found_settings.items():
+                _settings.append(protocol["settings"][v])
+            protocol["settings"] = _settings
+
```

Comment:
I think we should have some tests for this behavior.

Reply:
Done